src/parser/jobs/drk/changelog.tsx

```diff
@@ -6,6 +6,11 @@ export const changelog = [
 	// 	Changes: () => <>The changes you made</>,
 	// 	contributors: [CONTRIBUTORS.YOU],
 	// },
+	{
+		date: new Date('2025-05-23'),
+		Changes: () => <>Corrected reporting of Delirium windows that involve Impalement.</>,
+		contributors: [CONTRIBUTORS.AZARIAH],
```

Comment:
```suggestion
		contributors: [CONTRIBUTORS.VIOLET],
```

This should be crediting you, as the change maker, and you'll need to upload an avatar (max 100x100px / 100KB) to /src/data/avatar and add yourself to the /src/data/CONTRIBUTORS.ts file.

Reply:
Oh, whoops, this is a copy/paste error. Good catch, and I'll add an avatar.